src/ThirdPartySupport/Elementor/Widgets/V2/ElementorCampaignFormWidget/ElementorCampaignFormWidget.php

```diff
@@ -0,0 +1,261 @@
+        ]);
+
+        $this->add_control(
+            'display_style',
+            [
+                'label' => __('Display Style', 'give'),
+                'type' => \Elementor\Controls_Manager::SELECT,
+                'options' => ['onpage' => __('On Page', 'give'), 'modal' => __('Modal', 'give'), 'newTab' => __('New Tab', 'give')],
+                'default' => 'onpage',
+                'frontend_available' => true,
+            ]
+        );
+
+        $this->add_control(
+            'donate_button_text',
+            [
+                'label' => __('Donate Button Text', 'give'),
+                'type' => \Elementor\Controls_Manager::TEXT,
+                'default' => __('Continue to Donate', 'give'),
+                'frontend_available' => true,
+                'condition' => [
+                    'display_style!' => 'onpage',
+                ],
+            ]
+        );
+
+        $this->end_controls_section();
+    }
+
+    /**
+     * @unreleased
+     */
+    protected function getCampaignAndFormOptions(): array
+    {
+        $campaignsWithForms = $this->getCampaignsWithForms();
+
+        if (empty($campaignsWithForms)) {
+            return [[], []];
+        }
+
+        $campaignOptions = [];
+        $formOptionsGroup = [];
+        $campaignGroups = [];
+
+        // Group forms by campaign
+        foreach ($campaignsWithForms as $item) {
+            // Skip items without campaign association
+            if (empty($item->campaign_id) || empty($item->campaign_title)) {
+                continue;
+            }
+
+            $campaignId = $item->campaign_id;
+            $campaignTitle = $item->campaign_title;
+
+            // Add to campaign options if not already added
+            if (!isset($campaignOptions[$campaignId])) {
+                $campaignOptions[$campaignId] = $campaignTitle;
+                $campaignGroups[$campaignId] = [
+                    'label' => $campaignTitle,
+                    'options' => []
+                ];
+            }
+
+            // Add form to the campaign group
+            $campaignGroups[$campaignId]['options'][$item->id] = $item->title;
+        }
+
+        $formOptionsGroup = array_values($campaignGroups);
+
+        return [$campaignOptions, $formOptionsGroup];
+    }
+
+    /**
+     * @unreleased
+     */
+    protected function render(): void
+    {
+        $settings = $this->get_settings_for_display();
+        $campaignId = $settings['campaign_id'];
+        $displayStyle = $settings['display_style'];
+        $useDefaultForm = $settings['use_default_form'];
+        $donateButtonText = $settings['donate_button_text'];
+        $formId = $settings['form_id'];
+
+        if (empty($campaignId)) {
+            return;
+        }
+
+        echo do_shortcode(sprintf('[givewp_campaign_form campaign_id="%s" display_style="%s" use_default_form="%s" continue_button_title="%s" id="%s"]', $campaignId, $displayStyle, $useDefaultForm, $donateButtonText, $formId));
```

Comment:
Same thing here, I think we need to use the `esc_attr` method for these attributes.